src/reference/modules/administration/pages/procedures.adoc

```diff
@@ -265,24 +266,26 @@ It will also backup the application logs.
 
 ----
 
-# First, backup the LDAP database:
+# First, backup the LDAP database:
 /opt/rudder/sbin/slapcat -l /tmp/rudder-backup-$(date +%Y%m%d).ldif
 
 # Second, the PostgreSQL database:
 sudo -u postgres pg_dump rudder > /tmp/rudder-backup-$(date +%Y%m%d).sql
 
-# Or without sudo, use the rudder application password:
-pg_dump -U rudder rudder > /tmp/rudder-backup-$(date +%Y%m%d).sql
+# Or without sudo, use the rudder application password:
+pg_dump -Fc -U rudder rudder > /tmp/rudder-backup-$(date +%Y%m%d).sql
 
-# Third, backup the configuration repository:
-tar -C /var/rudder -zvcf /tmp/rudder-backup-$(date +%Y%m%d).tar.gz configuration-repository/ cfengine-community/ppkeys/
+# Third, backup the configuration repository:
+tar -C /var/rudder -zcf /tmp/rudder-backup-$(date +%Y%m%d).tar.gz configuration-repository/ cfengine-community/ppkeys/ packages/
```

Comment:
plugins folder too

Reply:
packages is the plugin folder

Reply:
ok!

Reply:
/var/rudder/plugin-resources contains plugins configuration